### ISSUE TITLE
Sidekick: add hlx3 support

### ISSUE THREAD

**Issue by rofe** [adobe/helix-pages#919]
- [ ] enable on `*.hlx3.page`
- [ ] use hlx3 reload
- [ ] use hlx3 publish ([example](https://github.com/adobe/spark-website/blob/main/tools/sidekick/plugins.js#L21-L87))

**Comment by rofe** [comment]
@trieloff on `*.hlx3.one` - I'm afraid don't really know what this is and how the sidekick is expected to behave in this environment. Please create a separate issue describing the desired Sidekick behavior.

**Comment by trieloff** [comment]
`*.hlx3.one` serves content just like `*.hlx3.page`, so the sidekick should be visible there. The reload/publish can still go through `admin.hlx3.page`

**Comment by rofe** [comment]
Yeah, well being visible is just the beginning. What about roundtrips to the editor and back to preview? How does the admin action know that it should use `hlx3.one` instead of `hlx3page`?

Just one of potentially multiple details to solve for. That's why I would really prefer a separate issue that describes the desired behavior for each sidekick action.
